### ISSUE TITLE
Assign priorities and pacing to our issues 

### ISSUE THREAD

**Issue by thornet** [mozilla/webmaker-curriculum#119]
There's a lot of filed issues (that's good). But it's hard to parse them and see when things are due. 

Could we get a scheme for prioritizing and pacing issues?

Proposal: 
* P1: must happen this "cycle"
* P2: should happen, or do if we have time. 
* [date]: heartbeat dates are probably best here, but sometimes we'll have other deadlines to hit. 

Happy to help sift and label.

**Comment by dajbelshaw** [comment]
+1 to this. I think that the main MoFo planning repo is/has moved to P1 and P2 using labels and heartbeats are indicated using milestones. I'll make a start with @LauraHilliger today. :)

**Comment by dajbelshaw** [comment]
OK, so I've:

* Created P1/P2 labels (same colour as at /plan)
* Changed and merged milestones to be named after heartbeats

Every issue has a milestone apart from a couple of things @LauraHilliger needs to review. All January 30 milestone issues have a P1/P2 tag except Issue #117 which is phrased as a question.

https://github.com/mozilla/webmaker-curriculum/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22January+30%22

**Comment by thornet** [comment]
Awesome! Thanks. Let's try out these labels and re-evaluate after a bit to see if we need to adjust.
